lib/commands/session.js

```diff
@@ -13,6 +13,7 @@ commands.createSession = async function (caps) {
   this.validateDesiredCaps(caps);
   this.sessionId = UUID.create().hex;
   this.caps = caps;
+  this.rawCaps = _.cloneDeep(caps); // used for reset
```

Comment:
Things would be easier to manage if we had session objects and the drivers were stateless handlers.